test/v2/init.js

```diff
@@ -24,38 +24,46 @@ var test = require('tape');
 var testRead = require('../lib/read_test.js');
 var Init = require('../../v2/init.js');
 
+/* jshint camelcase:false */
+
 var testInitReq = Buffer([
     0x00, 0x02,             // version:2
-
+    0x00, 0x03,             // nh:2
```

Comment:
ah right. i was assuming you were only sending the "supported" 2 headers and have been staring at hex all week. carry on.